README.md

```diff
@@ -273,6 +274,12 @@ db-migrate supports the concept of environments. For example, you might have a d
     "driver": "sqlite3",
     "filename": ":memory:"
   },
+	
+  "int": {
```

Comment:
what should int mean here? I think we should stick to dev, test and prod within the docu to avoid confusion